src/app/modules/content/content.state.ts

```diff
@@ -2,38 +2,37 @@ import { CONTENT_STORE_KEY } from './content.module';
 
 export interface ContentItem {
     id: number;
-    title: string;
-    url: string;
     imageUrl: string;
     newsSite: string;
-    summary: string;
     publishedAt: string;
+    summary: string;
+    title: string;
     updatedAt: string;
-}
-
-export type Report = ContentItem
```

Comment:
why did you removed extedning types for reports, blogs and articles? please revert

Reply:
OK I see it's moved down in the file